.github/workflows/js-lints.yml

```diff
@@ -0,0 +1,59 @@
+name: JavaScript Lints
```

Comment:
Minor change as this seems to also lint SCSS, perhaps this should read:

```suggestion
name: JavaScript and SASS Lints
```

Reply:
Why not "SCSS"?